demo/molecule/photoUploader/playground

```diff
@@ -0,0 +1,147 @@
+const demoStyles = {
+  margin:'auto',
+  marginTop: 20
+}
+
+const lineStyles = {
+  color: 'black',
+  paddingLeft: 20
+}
+
+const _maxPhotos = 10
+const _mainPhotoLabel = 'PRINCIPAL'
+const _addPhotoTextSkeleton = 'Añadir más'
+const _dropPhotosHere = 'Suelta las fotos aquí'
+const _addPhotoTextButton = 'Seleccionalas de tu dispositivo'
+const _dragPhotoTextInitialContent = 'Arrastra las fotos aquí'
+const _limitPhotosUploaded = 'Ha alcanzado el límite de fotografías'
+const _limitPhotosUploadedNotification = 'Máximo de fotografias alcanzado'
+const _errorFormatPhotoUploaded = 'Las fotografías deben tener formato JPEG, PNG, TIF o GIF'
+const _errorCorruptedPhotoUploaded = 'Archivo %{filepath} ha fallado'
+const _errorInitialPhotoDownloadError = 'Error al descargar imágenes'
+const _errorFileExcededMaxSize = 'Las fotografías deben tener un peso máximo de 50 MB'
+const _notificationErrorFormatPhotoUploaded = 'Solo se aceptanlos formatos: formato JPEG, PNG, TIF o GIF'
```

Comment:
This message is similar to `_errorFormatPhotoUploaded`

Reply:
Yes, but they also can be different.
These copies are all defined by props.
One is shown **BEFORE** the user enter the dropzone but not release the mouse button, and the other appears in the error notification bar, **AFTER** the user drops incompatible format on the dropzone.